src/content/guides/upgrade-tiptap-v2.mdx

```diff
@@ -138,6 +138,34 @@ new Editor({
 })
 ```
 
+### Introducing `shouldRerenderOnTransaction`
+
+The `@tiptap/react` Editor configuration now includes `shouldRerenderOnTransaction`, controlling component rerendering.
+
+Previously (pre-3.0.0), the Editor rerendered on every transaction-induced state update, facilitating continuous editor object tracking but potentially impacting rendering performance.
+
+Tiptap 3.0.0 enables `shouldRerenderOnTransaction` by default. Disable it by setting `shouldRerenderOnTransaction: false` in the editor configuration.
```

Comment:
Isn't it the opposite? Before tiptap 3, shouldRerenderOnTransaction was enabled by default. Now, it is disabled by default...